src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/lookups.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Use lookup table data in queries
+tags:
+  - Query your data
+  - 'NRQL: New Relic Query Language'
+  - NRQL query tutorials
+metaDescription: In New Relic, how to query data added via CSV lookup tables.
+redirects:
+---
+
+When you upload CSV-format lookup tables, you can use the `lookup()` function to access THAT data in your NRQL queries. 
+
+You can use lookups to make your telemetry data easier to understand, group data in custom ways, and include more information that wasn't available at the time for previously reported data. 
+
+## Upload a lookup table [#upload-table]
+
+For how to upload tables, see [Lookup tables](/docs/logs/ui-data/lookup-tables-ui). 
+
+## Query limitations [#limitations]
+
+You can't use lookup data in NRQL alert conditions. 
+
+The following NRQL functions are not supported with lookup queries: 
+
+* `TIMESERIES`
+* `COMPARE WITH`
+* `EXTRAPOLATE`
+* `JOIN` 
+
+However, these clauses can be used in an outer query where the inner query is a lookup query. For an example of this, see [this screenshot of a `JOIN` query](...??).
+
+## General query syntax [#general-syntax]
+
+Let's say you've named your table `storeNames`. The following query will select all data from that table.
+
+```
+FROM lookup(storeNames) 
+SELECT *
+```
+
+The following will select specific attributes from that same table: 
+
+```
+FROM lookup(storeNames) 
+SELECT store_ID, store_name, description
+```
+
+## Use lookup data with telemetry data [#query-with-data]
+
+The main benefit of using lookup tables is that you can use it with your New Relic-store telemetry data. For example, this query avoids hardcoding a long list of hosts by querying specific hosts of interest contained in a lookup table: 
+
+```
+FROM Log 
+SELECT count(*)
+WHERE hostname IN (FROM lookup(myHosts) SELECT uniques(myHost))
+```
+
+Using `JOIN` queries can make data more understandable. For instance, this query on a custom event type uses the `storeNames` table to show the store names along with the total sales. It also allows up to 10,000 stores, because lookup tables support a higher limit than other NRQL data types.
```

Comment:
```suggestion
Using `JOIN` queries can make data more understandable. For instance, this query on a custom event type uses the `storeNames` table to show the store names along with the total sales. It also allows up to 10,000 stores because lookup tables support a higher limit than other NRQL data types.
```